frontend/packages/dev-console/integration-tests/features/Knative/usability-Enhancement-Eventing-ODC-3953.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Usability Enhancement for Eventing
+    User should be able to select BootStrapServers and Topics from provided options
+
+
+Background:
+   Given cluster with OpenShift Serverless Operator installed
+   And Kafka Operator installed
+   And BootStrapServers and Topics created
+
+
+@regression, @smoke
+Scenario: BootStrapServers and Topics Drop Down
+    Given user is at Developer Perspective
+    When user goes to +Add page
+    And user clicks on Event Sources card
+    And user clicks on Kafka Sources
+    Then user will see the items in BootStrapServers dropdown
+    And user will see the items in Topics dropdown
+
+
+@regression, @smoke
+Scenario: Multiple BootStrapServers and Topics under their dropdown
+    Given user is at Developer Perspective
+    When user creates a BootStrapServer
+    And user creates a Topics
+    And user clicks on Event Sources card
+    And user clicks on Kafka Sources
+    Then user will see that server in BootStrapServers dropdown
+    And user will see that topic in Topics dropdown
+
+
+@regression, @smoke
+Scenario: Select multiple BootStrapServers and Topics
+    Given user is at Developer Perspective
+    When user creates a BootStrapServer
+    And user creates a Topics
+    And user clicks on Event Sources card
+    And user clicks on Kafka Sources
+    And user selects multiple BootStrapServers
+    And user selects multiple Topics
+    And user selects Sink
+    And user selects Application
+    And user enters name for the Kafka Source
+    And user clicks on Create button
+    Then user gets redirected to Topology page 
+    And user can see the Kafka source created
+
+
+@regression, @smoke
+Scenario: Select multiple BootStrapServers and Topics
+    Given user is at Developer Perspective
+    When user creates a BootStrapServer
+    And user creates a Topics
+    And user clicks on Event Sources card
+    And user clicks on Kafka Sources
+    And user enters the BootStrapServer name
+    And user enters the Topic name
+    And user selects Sink
+    And user selects Application
+    And user enters name for the Kafka Source
+    And user clicks on Create button
+    Then user gets redirected to Topology page 
+    And user can see the Kafka source created
```

Comment:
do we think we need different scenario text here `Select multiple BootStrapServers and Topics` as one is just selection and another one is typing

Reply:
One scenario is for selecting whatever we have in the drop down menu and the other is entering the bootstrapserver and topics. We can send the String there, that's why I have added the separate scenario there.